L74-java-channel-creds.md

```diff
@@ -0,0 +1,485 @@
+Java: Channel and Server Credentials
+----
+* Author(s): [Eric Anderson](https://github.com/ejona86)
+* Approver: sanjaypujare
+* Status: In Review {Draft, In Review, Ready for Implementation, Implemented}
+* Implemented in: <language, ...>
```

Comment:
Put Java here?

Reply:
It's not implemented yet, so I was leaving it as-is. I expected to update the gRFC after adding it to grpc-java to update this line. I'd also update the "Implementation" section to say which version it was added to.